modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/causalitydatanodes/DistributionZoneCausalityDataNodesTest.java

```diff
@@ -1535,15 +1575,17 @@ private WatchListener createMetastorageDataNodesListener() {
             for (EntryEvent event : evt.entryEvents()) {
                 Entry e = event.newEntry();
 
-                if (startsWith(e.key(), zoneDataNodesKey().bytes())) {
+                if (startsWith(e.key(), DISTRIBUTION_ZONE_DATA_NODES_HISTORY_PREFIX.getBytes(UTF_8))) {
```

Comment:
You already have byte representations as constants for these prefixes, why not use them here?

Reply:
fixed